src/components/scoresheets/scoresheets.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { fetchScoresheet, fetchScoresheets, removeScoresheet } from '../../actions/scoresheet_actions';
+
+//=========================================
+// props / actions
+
+const mapStateToProps = (state) => {
+  return {
+    scoresheets: state.scoresheets,
+    currentUser: state.auth.currentUser,
+  }
+};
+
+const mapDispatchToProps = (dispatch) => ({
+  fetchScoresheet: (scoresheetId) => dispatch(fetchScoresheet(scoresheetId)),
+  fetchScoresheets: (userId) => dispatch(fetchScoresheets(userId)),
+  removeScoresheet: (scoresheetId) => dispatch(removeScoresheet(scoresheetId)),
+});
+
+//=========================================
+// component
+
+class Scoresheets extends React.Component {
+
+  static propTypes = {
+
+  }
+
+  static defaultProps = {
+    scoresheet: { name: "LOADING..." }
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      scoresheets: props.scoresheets,
+      currentUser: props.currentUser,
+    }
+  }
+
+  componentDidMount() {
+    if (this.state.currentUser) {
+      this.fetchScoresheets(this.state.currentUser.id);
+    }
+  }
+
```

Comment:
This component will only be to fetch the current user's scoresheets? What about when I want to look at somebody else's scoresheets? I guess I'm not sure where this component will be going? What's the route for it on the frontend?

Reply:
Right - the route here currently is /scoresheets, which I assume would default to the current user's scoresheets. My thought is that another user's scoresheets would be displayed on something like a user/:id/scoresheets; certainly we'll need to change the logic here, but for now since we are dealing with just /scoresheets I figured this would be an okay default.

It seems to me the most sensible way is to pull the relevant id from mapStateToProps with another user's ID coming through in something like a UI slice of state, but as per my comment in the commit, that is currently pulling through as null (i.e. receive_user is only called after the component mounts, at least when loading /scoresheets directly).

Reply:
Ok, this will work for now, we will refactor when we make this component re-usable for other people's scoresheets